# Whole body control
verbose: false

# go1
# urdfPath: /root/code/go2_ws/src/legged_wbc/urdf/go1.urdf

# baseName: base
# contact3DofNames:
#   - LF_FOOT
#   - RF_FOOT
#   - LH_FOOT 
#   - RH_FOOT

# contact6DofNames: []

# go2
urdfPath: /root/code/go2_ws/src/legged_wbc/urdf/go2.urdf

baseName: base
contact3DofNames:
  - FL_foot
  - FR_foot
  - RL_foot 
  - RR_foot

contact6DofNames: []

baseAccelTask:
  baseAcc_kp:
    - 0.0       # p_base_x
    - 0.0       # p_base_y
    - 100.0       # p_base_z
    - 0.0       # 
    - 0.0       # 
    - 0.0       # 
  baseAcc_kd:
    - 0.0      # vx
    - 0.0      # vy
    - 0.1      # vz
    - 0.0      # wx
    - 0.0      # wy
    - 0.0      # wz

swingLegTask:
  kp: 350
  kd: 37

torqueLimitsTask:
  - 33.5     # HAA
  - 33.5     # HFE
  - 33.5     # KFE

frictionConeTask:
  frictionCoefficient: 0.3

# WeightedWbc
weight:
  baseAccel: 1
  contactForce: 0.01
  swingLeg: 0
  jointTorque: 0

# joint PD
jointKp: 10
jointKd: 0.5

